src/audio.c

```diff
@@ -715,6 +715,20 @@ static void ausrc_error_handler(int err, const char *str, void *arg)
 }
 
 
+struct telev_work {
+	struct audio *a;
+	struct mbuf *mb;
+};
```

Comment:
if there is going to be alot of these types, it might be better to define one struct
with event type and event payload union. example:

```c
struct work {
    enum event_type type;
    union {
        struct {
            int foo;
        } telev;
        struct {
            char bar;
        } pt;
   } u;
};
```